.travis.yml

```diff
@@ -2,18 +2,23 @@ language: generic
 matrix:
   include:
     - script:
-        - xcodebuild -scheme Yams test
+        - xcodebuild -scheme Yams build-for-testing test-without-building # -sdk macosx -destination arch=x86_64
```

Comment:
That was workaround for unstable testing on Xcode 8.
I don't know whether the workaround is still needed or not anymore.
https://github.com/Quick/Quick/commit/ed9c17b8c39a5a36d0244a2e1ff4450a4b971a30